src/main.rs

```diff
@@ -127,11 +161,18 @@ fn pg_client(options: &Options) -> Result<Client, Box<dyn Error>> {
 
 fn main() -> Result<(), Box<dyn Error>> {
     let options = Options::load()?;
+
     let mut client = pg_client(&options)?;
 
     // Restrict `search_path` to just the one schema.
     client.execute(&format!("SET SCHEMA {}", options.schema.sql_value()), &[])?;
     client.execute("BEGIN ISOLATION LEVEL REPEATABLE READ READ ONLY;", &[])?;
+    if !&options.features.is_empty() {
+        let vec: Vec<String> = options.features.clone().into_iter().collect();
+        let array_literal = format!("{{{}}}", vec.join(","));
+        client.execute(&format!("SET pg_parcel.features = '{array_literal}'"), &[])?;
```

Comment:
Let's say that in addition to `pg_parcel.features` we also had `pg_parcel.feature.{feature}`, then we could have queries like:

```sql
select
...
where current_setting('pg_parcel.feature.foobar', true) is not null
```

(The `true` means a missing var is okay, and `null` is returned.)

Reply:
And the value here is that the queries are a bit simpler? I can see how this might be a nice option.

Reply:
Yeah, but looking at it now, maybe it muddies the water.